ttl/hbp_scale_ontology.ttl

```diff
@@ -54,7 +54,7 @@ HBP_SCL:0000005 rdf:type owl:Class ;
 
 HBP_SCL:0000006 rdf:type owl:Class ;
                  rdfs:label "single cell"@en ;
-                 nsu:definition "One cell."@en ;
+                 nsu:definition "One cell, one neuron."@en ;
```

Comment:
is it a single neuron or single cell ? If it is a neuron, then align with the rdfs:label. It can be reformulate as "single neuron".